src/handlers/thirdparty.ts

```diff
@@ -34,4 +36,37 @@ export function handleThirdPartyAdded(event: ThirdPartyAdded): void {
 
   let metric = buildCountFromThirdParty()
   metric.save()
-}
\ No newline at end of file
+}
+
+export function handleItemUpdated(event: ItemUpdated): void {
+  let item = Item.load(event.params._itemId)
+  if (item == null) {
+    return
+  }
+
+  item.metadata = event.params._metadata
+
+  item.save()
+}
+
+export function handleItemAdded(event: ItemAdded): void {
+  let collectionId = getCollectionId(event.params._itemId)
+  let itemId = getItemId(event.params._itemId)
```

Comment:
the `itemId` must be `thirdpartId-itemId`. We can have duplicated values if we only use the itemId